interfaces/cssom.idl

```diff
@@ -123,6 +128,18 @@ interface CSSStyleDeclaration {
   [CEReactions] attribute [TreatNullAs=EmptyString] CSSOMString cssFloat;
 };
 
+partial interface CSSStyleDeclaration {
+  [CEReactions] attribute [TreatNullAs=EmptyString] CSSOMString _camel_cased_attribute;
```

Comment:
This is wrong, given this is actually some magic variable thing.

Reply:
@foolip - These aren't really part of the spec; do we have (or, need) a mechanism for excluding example/descriptive snippets of IDL from being scraped?

Reply:
I think this might be a CSSOM-only oddity? Most other places the web platform does such things we rely on named getters, hence it might not be necessary to have a general solution to this problem?

Reply:
IDL in examples is already excluded, but this is a macro of sorts. I guess what we should do is to generate the IDL that it expands to just like how implementations should and Gecko in fact does.

But just excluding this snippet with some string matching hack should be a good enough start.

Reply:
@mdittmer - I'll delegate to the scraper tool to ignore these chunks.

Reply:
@gsnedders, if this isn't *really* IDL, would it be possible to mark it with the `note`, or even `example` CSS class?